docker/openchallenges/services/mariadb.yml

```diff
@@ -5,9 +5,9 @@ services:
     restart: always
     env_file:
       - ../../../apps/openchallenges/mariadb/.env
-    volumes:
-      # - openchallenges-mariadb:/data/db
-      - ../../../apps/openchallenges/mariadb/docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
+    # volumes:
```

Comment:
shouldn't we at least mount and empty volume to persist data?

Reply:
It's not needed during development.